signature/src/policy.rs

```diff
@@ -0,0 +1,460 @@
+// Copyright (c) 2021 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::{anyhow, Result};
+use oci_distribution::Reference;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::convert::TryFrom;
+use std::fs;
+use std::vec::Vec;
+use std::str::FromStr;
+
+use crate::image;
+use crate::signatures;
+
+#[derive(EnumString, Display, Debug, PartialEq)]
+pub enum PolicyReqType {
+    #[strum(to_string = "insecureAcceptAnything")]
+    TypeAccept,
+    #[strum(to_string = "reject")]
+    TypeReject,
+    #[strum(to_string = "signedBy")]
+    TypeSignedBy,
+}
+
+#[derive(EnumString, Display, Debug, PartialEq)]
+pub enum KeyType {
+    #[strum(to_string = "GPGKeys")]
+    Gpg,
+}
+
+#[derive(EnumString, Display, Debug, PartialEq)]
+pub enum PolicyReqMatchType {
+    #[strum(serialize = "matchExact")]
+    TypeMatchExact,
+    #[strum(serialize = "matchRepoDigestOrExact")]
```

Comment:
I would expand "Repo" to "Repository" so it is kept consistent with the other match types.

Reply:
The field names of the policy file now follow [the definitions in containers/image](https://github.com/containers/image/blob/main/docs/containers-policy.json.5.md#signedby), and `MatchRepositoryDigestOrExact` is a little long as a variable name :)

Reply:
ah, ok. Could you add this link somewhere on this module? So reviewers like me realize this module implements a standard.

Reply:
OK, I have add the link at the comment of `Policy` struct :)